cabal.project

```diff
@@ -57,6 +61,8 @@ constraints:
   , singletons < 3.0
   -- breaks eventful even more than it already was
   , persistent-template < 2.12
+  -- needed by playground-common
+  , recursion-schemes < 5.2
```

Comment:
If it's in a cabal file we control, we might as well put it there! (i.e. just put this in `playground-common.cabal`)